README.md

```diff
@@ -17,74 +26,167 @@ capacity, the more memory you will use.
 You may construct the Bloom filter capable of receiving 1 million elements with a false-positive
 rate of 1% in the following manner. 
 
-```Go
-    filter := bloom.NewWithEstimates(1000000, 0.01) 
+```python
+from shaped_bloom_filter import BloomFilter
+filter = BloomFilter(1000000, 0.01)
+```
+
+Operations like the following can be done:
+
+```python
+# single member addition
+assert filter.is_member(10) == False
+filter.add(10)
+assert filter.is_member(10) == True
+
+# multiple member addition
+assert filter.are_members([1, 5, 9]) == [0, 0, 0]
+filter.add_batch([5, 9])
+assert filter.are_members([1, 5, 9]) == [0, 1, 1]
+
+# serialization
+serialized = filter.serialize()
+new_filter = BloomFilter(restore_from_serialized=serialized)
+assert new_filter.are_members([1, 5, 9]) == [0, 1, 1]
+```
+
+For composing your own keys, you can use the following method:
```

Comment:
not sure what you mean by "composing your own keys"

Reply:
"For composing your own keys using Pythonic objects ..."